Sources/ComposedUI/TableView/TableCoordinator.swift

```diff
@@ -304,6 +303,14 @@ extension TableCoordinator: SectionProviderMappingDelegate {
         tableView.moveRow(at: sourceIndexPath, to: destinationIndexPath)
     }
 
+    public func mappingDidInvalidateHeader(at sectionIndex: Int) {
+        // TODO: Reload header
+    }
+
+    public func mappingDidInvalidateFooter(at sectionIndex: Int) {
+        // TODO: Reload header
```

Comment:
A future PR can cover the updates already made to `UICollectionView`, including batched updates.